src/game-support/trackmania.md

```diff
@@ -4,9 +4,9 @@
 {{#template ../templates/rating.md status=Silver installs=Yes opens=Yes}}
 
 > [!NOTE]
-> On MacOS Sonoma the games launch and finishing a map is technically possible, but pratically nothing is rendered (only trees ?)
+> On MacOS Sonoma (before GPTK 2) the game launch and finishing a map is technically possible, but pratically nothing is rendered properly
```

Comment:
Can you clarify on what you mean about "MacOS Sonoma (before GPTK 2)"? 

Are you saying that you've:
A) only tested the game on MacOS Sonoma, and before the GPTK 2.0 update?
or B) tested the game on both MacOS Sonoma and MacOS Sequoia, both before and after the GPTK 2.0 update, and this issue only occurs with Sonoma before the 2.0 update?

If its A, you can remove this clarification about MacOS Sonoma, and I request that you test the game with GPTK 2.0, and if it's B, can you add a clarification about how the game runs with Sequoia and GPTK 2.0?

Reply:
Well the game launch before GPTK 2.0 but practically nothing renders (some trees), but I managed to end multiple races because I know them, so the game logic works, but there's no render, and with GPTK 2.0 everything works as intended.

Reply:
ahh i see what you're saying, i'm gonna test it myself to see what you're saying then this can get merged, though i'll probably remove the note about GPTK 2 because we don't typically include any statements that affect previous versions of whisky

Reply:
@hahayupgit I think you can write this idea down - about skiping info about previous Whisky versions in docs. I agree with that.